revpi_core.c

```diff
@@ -99,6 +99,7 @@ u8 revpi_core_find_gate(struct net_device *netdev, u16 module_type)
 		return piCore_g.i8uLeftMGateIdx;
 	}
 
+ret_undef:
```

Comment:
Please remove this now unused label (I wonder if there was no warning at compilation)